sentry/src/lib.rs

```diff
@@ -2,13 +2,24 @@
 #![deny(rust_2018_idioms)]
 
 use hyper::service::{make_service_fn, service_fn};
-use hyper::{Body, Error, Request, Response, Server, StatusCode};
+use hyper::{Body, Error, Method, Request, Response, Server, StatusCode};
 use primitives::adapter::Adapter;
 use primitives::Config;
 use slog::{error, info, Logger};
 
 pub mod routes {
     pub mod channel;
+    pub mod cfg {
+        use crate::ResponseError;
+        use hyper::{Body, Response};
+        use primitives::Config;
+
+        pub fn return_config(config: &Config) -> Result<Response<Body>, ResponseError> {
+            let config_str = serde_json::to_string(config)?;
+
+            Ok(Response::builder().body(Body::from(config_str)).unwrap())
```

Comment:
This is missing the `application/json` response content type

Reply:
Good catch!